beast/plotting/plot_toothpick_details.py

```diff
@@ -1,13 +1,16 @@
 import matplotlib.pyplot as plt
+import tables
```

Comment:
It is critical to use tables?  This package is not the greatest python package and I have been slowly trying to remove it from the BEAST to avoid using it.  I have found I can replace this package with astropy.tables and/or h5py.